### DISCUSSION TITLE
话说有人明天考CSP-J/S吗

### DISCUSSION THREAD

**Discussion by funnyyyyyyyy** [share121/inter-knot#5241] [科技]
不知道的可以去搜搜 信息学奥赛

**Comment by cuijiaqi0406** [comment]
刚看完c语言的成绩就看到这个<img src="https://bbs-static.miyoushe.com/static/2023/11/16/9869e04ba6a0d0e6ed2418efc27d4b74_4411752698018709526.png" alt="苍角-诶？" width="75" >

**Comment by Aha-follower** [comment]
好怀念，我们当时是第一届csp非专业组，现在acm都退役了。
